backend/app/core/ipfs.py

```diff
@@ -0,0 +1,32 @@
+import ipfshttpclient
+
+from ipaddress import ip_address, ip_interface, IPv4Address, IPv6Address
+
+from.config import IPFS_ADDRESS, IPFS_PORT
```

Comment:
relative import? The file is on the same directory right?

Reply:
The `.env` file is not in the same directory, but the package that parse the `.env` is there. See `backend/app/core/config.py`. 

However there should be  a space between `from` and `.config`, I'll fix that.